indico/modules/events/management/controllers/cloning.py

```diff
@@ -125,12 +139,14 @@ def _process(self):
         form = CloneRepeatabilityForm()
         clone_calculator = get_clone_calculator(form.repeatability.data, self.event)
         try:
-            dates = clone_calculator.calculate(request.form)
+            result = clone_calculator.calculate(request.form)
```

Comment:
I would return a tuple instead of a dict:

    dates, last_day_of_month = clone_calculator.calculate(request.form)